### PR TITLE
feat: thorsavers mixpanel events

### PR DESCRIPTION
**Pull request by reallybeard**
## Description

Add events for
- Deposit
- Withdraw

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

closes #

## Risk

Small risk

### Operations

Test thorchain savers operate as normal.

### CODE REVIEW

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx [hunk 1]
```diff
@@ -221,6 +225,11 @@ export const Deposit: React.FC<DepositProps> = ({
         })
         onNext(DefiStep.Confirm)
         contextDispatch({ type: ThorchainSaversDepositActionType.SET_LOADING, payload: false })
+        mixpanel?.track(MixPanelEvents.DepositContinue, {
+          provider: opportunityData.provider,
+          type: opportunityData.type,
+          assets: opportunityData.underlyingAssetIds.map(getCompositeAssetSymbol),
+        })
```
Comment (gomesalexandre) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx:232]: Let's add the human crypto amount also 🙏🏽 and fiat amount if you also need it

File: src/pages/Defi/components/OpportunityCard.tsx [hunk 2]
```diff
@@ -80,6 +84,12 @@ export const OpportunityCard = ({
 
   const handleClick = () => {
     if (isConnected) {
+      mixpanel?.track(MixPanelEvents.ClickOpportunity, {
+        provider,
+        type,
+        assets: [getCompositeAssetSymbol(underlyingAssetId ?? '')],
```
Comment (gomesalexandre) [inline src/pages/Defi/components/OpportunityCard.tsx:90]:
```suggestion
        assets: [getCompositeAssetSymbol(underlyingAssetId)],
```

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx [hunk 3]
```diff
@@ -253,6 +253,10 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
           provider: opportunityData.provider,
           type: opportunityData.type,
           assets: opportunityData.underlyingAssetIds.map(getCompositeAssetSymbol),
+          fiatAmounts: [bnOrZero(state?.withdraw.fiatAmount).toNumber()],
+          cryptoAmounts: [
+            `${state?.withdraw.cryptoAmount} ${getCompositeAssetSymbol(assetId ?? '')}`,
```
Comment (gomesalexandre) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx]:
```suggestion
            `${state?.withdraw.cryptoAmount} ${getCompositeAssetSymbol(assetId)}`,
```

File: src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Status.tsx [hunk 4]
```diff
@@ -97,13 +97,18 @@ export const Status: React.FC<StatusProps> = ({ accountId }) => {
         provider: opportunity?.provider,
         type: opportunity?.type,
         assets: opportunity?.underlyingAssetIds.map(getCompositeAssetSymbol),
+        fiatAmounts: [bnOrZero(state.withdraw.fiatAmount).toNumber()],
+        cryptoAmounts: [`${state.withdraw.cryptoAmount} ${getCompositeAssetSymbol(assetId ?? '')}`],
```
Comment (gomesalexandre) [inline src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Status.tsx:101]:
```suggestion
        cryptoAmounts: [`${state.withdraw.cryptoAmount} ${getCompositeAssetSymbol(assetId)}`],
```

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**